spec/schemas/_common.query_dsl.yaml

```diff
@@ -1248,8 +1248,6 @@ components:
           type: number
         filter:
           $ref: '#/components/schemas/QueryContainer'
-      required:
-        - model_id
```

Comment:
Is model ID not required? IMO it is.

Reply:
Per [documentation](https://opensearch.org/docs/latest/query-dsl/specialized/neural/), it is required if the default model id is not set. [Here](https://opensearch.org/docs/latest/query-dsl/specialized/neural/) is explained how to configure it.